dox_trace/spec/test_input/verification_methods/index.rst

```diff
@@ -54,8 +54,7 @@ Backward
 --------
 
 .. spec:: SWA_Spec_VerificationMethodsBackward
-    :test_setups: off_target, on_target
+    :verification_methods: off_target, on_target
 
 .. spec:: SWA_Spec_VerificationMethodsBackwardBoth
```

Comment:
Seems some tests can be removed now.
There might be also some requirement IDs which are not needed anymore like "DoxTrace_Export_TestSetups".

I assume a few tests can be cleaned up.

Reply:
I did remove few very easy to understand test cases, but did not remove them all fearing I may remove some important logic..
But I will take a look once again. Thanks